### DISCUSSION TITLE
`IParameterSymbol.GetDocumentationCommentXml` for primary constructor parameters always return `string.Empty`

### DISCUSSION THREAD

**Discussion by kyoyama-kazusa** [dotnet/roslyn#67858] [Q&A]
I don't know whether this is a bug ... When I use incremental generator to generate for members for primary constructor parameters, I cannot fetch XML doc comments:

```csharp
var comment = parameterSymbol.GetDocumentationCommentXml(expandIncludes: true, cancellationToken: ct);
```

I ensure the variable `parameterSymbol` has a correct value, but I don't know why `comment` is `""`.

![](https://user-images.githubusercontent.com/23616315/232790799-accbe015-f061-4cfe-b6fc-8e6cff50e760.png)

Generated source code:

![](https://user-images.githubusercontent.com/23616315/232791338-6cc36be5-7296-4b8a-a402-92770a4e2829.png)

The doc comments of generated members are the default one I have set:

![image](https://user-images.githubusercontent.com/23616315/232791624-005c1c43-696d-4f26-a28e-1fb3096cda18.png)

**Comment by RikkiGibson** [comment]
This looks like a bug in Roslyn. If `GetDocumentationCommentXml` is returning a meaningful value for a parameter like `str` below, I think we can be pretty sure that is the case.

```cs
class C
{
    /// <param name="str">The parameter</param>
    void M(string str) { }
}
```

**Reply by kyoyama-kazusa** [reply]
Thank you. I'll open an issue for this topic.
